src/common/verify/isValidAccessGrant.test.ts

```diff
@@ -112,6 +112,21 @@ describe("isValidAccessGrant", () => {
     expect(mockedFetch).toHaveBeenCalled();
   });
 
+  it("uses the provided fetch if any from plain JSON VC", async () => {
+    jest.mocked(isVerifiableCredential).mockReturnValueOnce(true);
+    const mockedFetch = jest.fn<typeof fetch>().mockResolvedValueOnce(
+      new Response(JSON.stringify(MOCK_VERIFY_RESPONSE), {
+        status: 200,
+      }),
+    );
+    await isValidAccessGrant(JSON.parse(JSON.stringify(MOCK_ACCESS_GRANT)), {
```

Comment:
```suggestion
    await isValidAccessGrant(MOCK_ACCESS_GRANT_BASE), {
```
Although what you had isn't incorrect, since `MOCK_ACCESS_GRANT` is actually derived from `MOCK_ACCESS_GRANT_BASE`, so we might as well use it directly.